frontend/src/screens/apps/ShowApp.tsx

```diff
@@ -222,7 +222,7 @@ function AppInternal({ app, refetchApp, capabilities }: AppInternalProps) {
             <CardHeader>
               <CardTitle>Info</CardTitle>
             </CardHeader>
-            <CardContent>
+            <CardContent className="slashed-zero">
```

Comment:
can you be more precise where you add it? this adds the classname to all child elements

Reply:
I wanted the Dates and Pubkey etc to also have a slashed zero, so I applied it to the parent like we did in the DialogTrigger in TransactionItem
https://github.com/getAlby/hub/blob/71183a131a03919f98ee9db271353ab52f1021ba/frontend/src/components/TransactionItem.tsx#L164